pkgs/development/libraries/qt-5/modules/qtwebkit.nix

```diff
@@ -23,15 +23,15 @@ let
 in
 qtModule {
   name = "qtwebkit";
-  qtInputs = [ qtbase qtdeclarative qtlocation qtsensors ] ++ optionals (lib.versionAtLeast qtbase.version "5.11.0") [ qtwebchannel ];
+  qtInputs = [ qtbase qtdeclarative qtlocation qtsensors ] ++ optionals (lib.versionAtLeast qtbase.version "5.11.1") [ qtwebchannel ];
```

Comment:
You don't need to modify these lines, it should still work, since 5.11.1 > 5.11.0.

Reply:
Oops. I thought I had build errors before, but apparently that was unrelated. Reverted and rebased.